test/e2e-tests.sh

```diff
@@ -16,29 +16,109 @@
 
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
+TMPF=$(mktemp /tmp/.mm.XXXXXX)
+clean() { rm -f ${TMPF}; }
+trap clean EXIT
+
+
 # Setup a test cluster.
-[[ -z ${LOCAL_CI_RUN} ]] && initialize $@
+[[ -z ${LOCAL_CI_RUN} ]] && {
+    # Initialize cluster
+    initialize $@
+
+    # Install the latest Tekton CRDs.
+    kubectl apply --filename https://storage.googleapis.com/tekton-releases/latest/release.yaml
+}
+
+# Allow ignoring some yaml tests, space separated, should be the basename of the test
+#
+# Disabled by default:
+#
```

Comment:
I'm a bit confused by this comment, maybe there could be more detail in the comment? sounds like some yaml tests have a name that doesn't make sense and so they are being skipped?